lib/vec.h

```diff
@@ -1,6 +1,6 @@
 /* -*- C -*- */
 /*
- * Copyright (c) 2013-2020 Seagate Technology LLC and/or its Affiliates
+ * Copyright (c) 2017-2021 Seagate Technology LLC and/or its Affiliates
```

Comment:
It's OK to update this.
But if it is not a big or important change, no need to update it.

Many files are modified by this patch. Not all of them are updated in the copyright section.